### PR TITLE
[orchestrator] Makes the web static files path configurable.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: conf.toml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+
 [AccountManager]
 Type = "unix"
 
 [InstanceManager]
 Type = "unix"
 
 [InstanceManager.GCP]
-ProjectId = ""
+ProjectID = ""
```
Comment (jemoreira) [inline conf.toml:10]: This doesn't seem related to this change

File: pkg/app/signalingserver.go [hunk 2]
```diff
@@ -32,11 +32,15 @@ import (
 // communicating with the host orchestrator in the appropriate instance and
 // forwarding all requests to it.
 type ForwardingSignalingServer struct {
+	interceptPath   string
```
Comment (jemoreira) [inline pkg/app/signalingserver.go:35]: nit: `interceptedFilesPath` is more clear

Comment (ser-io) [reply pkg/app/signalingserver.go]: used `connectorStaticFilesPath` as discussed offline.

File: conf.toml (hunk 1 above)
Comment (ser-io) [reply conf.toml]: true, reverted.
